include/SFML/Audio/SoundRecorder.h

```diff
@@ -132,5 +134,58 @@ CSFML_AUDIO_API sfBool sfSoundRecorder_isAvailable(void);
 ////////////////////////////////////////////////////////////
 CSFML_AUDIO_API void sfSoundRecorder_setProcessingInterval(sfSoundRecorder* soundRecorder, sfTime interval);
 
+////////////////////////////////////////////////////////////
+/// \brief Get a list of the names of all availabe audio capture devices
+///
+/// This function returns an array of strings, containing
+/// the names of all availabe audio capture devices.
```

Comment:
You should state that the returned strings are null-terminated, this is important in C (this applies to the other functions as well).